abel/analytical.py

```diff
@@ -1,11 +1,70 @@
 # -*- coding: utf-8 -*-
 import numpy as np
+from abel.tools import index_coords, cart2polar
 
 
 # This file includes functions that have a known analytical Abel transform.
-# They are used in unit testing as well for comparing different iAbel impementations.
+# They are used in unit testing as well for comparing different Abel impementations.
 # See BaseAnalytical class for more information.
 
+def sample_image_dribinski(n=361, origin=None):
+    """
+    Sample test image used in the BASEX paper Rev. Sci. Instrum. 73, 2634 (2002) 
+    9x Gaussian functions of half-width 4 pixel + 1 background width 3600
+      - anisotropy - ß = -1  for cosθ term
+                   - ß = +2  for sinθ term
+                   - ß =  0  isotropic, no angular variation
+    (there are some missing negative exponents in the publication)
+
+    Parameters
+    ----------
+     - n: integer (square) image width (height)
+     - origin: tuple,  (row, col) center of image
+
+    Returns
+    -------
+     - IM: 2D n x n numpy array image
+
+    """
+
+    def Gauss (r, r0, sigma2):
+        return np.exp(-(r-r0)**2/sigma2)
+
+    def I(r, theta):  # intensity function Eq. (16)
+        t0 = 7*Gauss(r, 10, 4)*np.sin(theta)**2
+        t1 = 3*Gauss(r, 15, 4)
+        t2 = 5*Gauss(r, 20, 4)*np.cos(theta)**2
+    
+        t3 =   Gauss(r, 70, 4)
+        t4 = 2*Gauss(r, 85, 4)*np.cos(theta)**2
+        t5 =   Gauss(r, 100, 4)*np.sin(theta)**2
+
+        t6 = 2*Gauss(r, 145, 4)*np.sin(theta)**2
+        t7 =   Gauss(r, 150, 4)
+        t8 = 3*Gauss(r, 155, 4)*np.cos(theta)**2
+
+        t9 = 20*Gauss(r, 45, 3600)  # background under t3 to t5
+    
+        return 2000*(t0+t1+t2) + 200*(t3+t4+t5) + 50*(t6+t7+t8) + t9
+
+    # set up an dummy image 2D n x n numpy array shape 
+    im = np.arange(n*n).reshape((n,n))
+
+    if origin is None:
+        origin = (n//2+n%2, n//2+n%2)
+
+    # determine cartesian coordinates (x,y), for each pixel
+    x, y = index_coords(im, origin=origin)
```

Comment:
It might be more straightforward to use meshgrid, since that's basically what you are setting up. (Also, I'm not sure if this is as real python style, but I tend to use lower-case ``x`` for variables or 1D arrays, and ``X`` for 2D arrays.) You could use:

```python
x = np.arange(n)
X,Y = np.meshgrid(x,x)
R,THETA = cart2polar(X,Y) 
IM = I(R,THETA)